tests/builders/test_task_config.py

```diff
@@ -1,22 +1,24 @@
 import warnings
+from typing import Tuple
 
 import torch
-from hypothesis import given
 from myrtlespeech.builders.task_config import build
+from myrtlespeech.model.seq_to_seq import SeqToSeq
 from myrtlespeech.protos import task_config_pb2
 
-from tests.protos.test_task_config import task_configs
+# Utilities -------------------------------------------------------------------
 
 
-# Tests -----------------------------------------------------------------------
-
-
-@given(task_cfg=task_configs())
-def test_build_returns(task_cfg: task_config_pb2.TaskConfig) -> None:
-    """Test that build returns when called."""
-    model, epochs, train_loader, eval_loader = build(task_cfg)
-    assert isinstance(model, torch.nn.Module)
```

Comment:
It's probably worth keeping this test such that there is a minimal test case for building the task config? It's duplicated in the Saver test but this could be modified in future or fail in any of the later parts after the build.

Reply:
Ok I've added back in (but keeping the helper function)